tests/builtins/test_isinstance.py

```diff
@@ -70,6 +67,24 @@ class MyClass2(MyClass1):
             print(isinstance(obj, MyClass2))
             """, run_in_function=False)
 
+    def test_subclass_grandparent(self):
```

Comment:
Maybe should be called `test_isinstance_grandparent`? A similar test should perhaps than also be built for `issubclass(MyClass3, MyClass1)`.